index.html

```diff
@@ -244,9 +269,71 @@
         </dt>
       </dl>
       <p>
-        The <code>PermissionName</code> enum defines the list of known
-        permission names.
+        Each enumerator-value in the <a>PermissionName</a> enum identifies a
+        <dfn data-export="">permission</dfn>, which consists of the following
+        algorithms and types:
       </p>
+      <dl>
+        <dt>
+          An <dfn data-export="">associated PermissionDescriptor</dfn>
```

Comment:
I'd actually lean toward "permission query type", but "associated PermissionDescriptor" was already defined in the file, and that cleanup seemed like something I could plausibly leave to a subsequent PR. This one's already huge.